.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
```

Comment:
Suggestion: if we aren't testing against multiple versions, I feel using a matrix is a bit excessive: we could directly specify the version. Additionally, could we version 20 (currently active LTS) to match our [deployment workflow](https://github.com/designatuci/DUCI-website/blob/main/.github/workflows/deploy-main.yml)?